utils.js

```diff
@@ -399,6 +399,20 @@ function formatReadReceipt(event) {
   };
 }
 
+function formatRead(event) {
+  return {
+    tids: event.tids,
```

Comment:
Hello, so what key name should I change from ```tids``` ? ```threadIDs``` ?
the "tids" looks like => tids: [ 'id.222346744472064' ]
I'm not sure whether it is thread IDs or not.
All the keys here are simply as same as Facebook's ```event``` object.
It would be nice that you can give me some suggestions for the naming of keys .
Thanks.